src/server/seeder/index.ts

```diff
@@ -0,0 +1,16 @@
+import "reflect-metadata";
+import { createConnection } from "typeorm";
+import { Merchant } from "../entity/Merchant";
+const { merchants } = require("../mockMerchantData")
```

Comment:
I'm more used to es modules and my IDE is set to default for those as well, ideally it should all be unified to either use commonsJS or esModules